api/src/api_models/datasource.ts

```diff
@@ -3,6 +3,26 @@ import { Length, IsString, IsOptional, ValidateNested, IsUUID, IsIn, IsInt, IsOb
 import { Type } from 'class-transformer';
 import { Authentication, FilterRequest, PaginationRequest, PaginationResponse, SortRequest } from './base';
 
+@ApiModel({
+  description: 'Authentication object for using app_id and app_secret',
+  name: 'Authentication',
```

Comment:
Name of the ApiModel should be equal to the class name. In this case it should be DataSourceProcessingConfig

Reply:
Also update the description